src/MoBi.Assets/AppConstants.cs

```diff
@@ -2268,11 +2269,21 @@ public static class PKSim
                                                                       "After changing the PK-Sim installation path, please restart MoBi.";
       }
 
+      public static string DefaultFileNameForBuildingBlockExport(string projectName, string simulationName)
```

Comment:
The `simulationName` is probably not a simulation but the name of the building block so that should be renamed